javascript/javascript1/week2/other_task.js

```diff
@@ -0,0 +1,176 @@
+
+
+// Event application
+function getEventWeekday(day){
+  const weekDays=['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const currentDate  = new Date();
+  const dayNumbers = currentDate.getDay();
+   const futureDay= (day + dayNumbers) % 7;
+   console.log(weekDays[futureDay]);
+}
+getEventWeekday(1);
+
+
+// Weather wear 
+ function SuitableClothes(temperture){
```

Comment:
Good use of if-else-if-else! ⭐ The function name can be misleading though! Many will likely expect something else when seeing the starting letter capitalized; you can even see that GitHub renders it differently. 👀

This is all convention though as has no functional impact in JavaScript. My suggestion is to stick to standard conventions or at least be consistent within your project, so to stick with convention and be consistent with your other functions, I'd call this function `suitableClothes`. 😄 

I would not recommend your alternative `switch`-based approach as it is a lot more error prone and generally harder to maintain due to the weird, different syntax. 🥴